packages/nextjs/test/integration/next-env.d.ts

```diff
@@ -1,4 +1,5 @@
 /// <reference types="next" />
+/// <reference types="next/types/global" />
```

Comment:
What is this change about?

Reply:
Hmm good catch, that must have slipped into the commit. I didn't change it though, maybe that's changed when running NextJS tests (which I did) 🤔 
Anyway, I'll remove it